todo-with-kuzzle-and-vueJS/README.fr.md

```diff
@@ -0,0 +1,601 @@
+ - `Task.vue` => ligne correspondant à une tache avec boutons pour compléter/supprimer.
+
+La construction de ces composants ne sera pas détaillée étant donné qu'ils n'ont aucune intéraction directe avec Kuzzle. Vous pouvez vous référer aux fichiers correspondants pour avoir un exemple de code.
+
+### Add
+Créez le composant `/src/components/Add.vue`
+Ce composant est assez simple. La partie template sera composée d'un champ de saisie de texte ainsi que d'un bouton. Lors du click, un signal `addTask` sera envoyé au parent de ce composant, accompagné de la saisie.
+
+### ModalList
+Créez le composant `/src/components/ModalList.vue`
+Ce composant doit contenir un champ de saisie de text ainsi qu'un bouton. Lors d'un click, un signal `create` sera envoyé au parent de ce composant, accompagné de la saisie.
+
+### ManageList
+Créez le composant `/src/components/ManageList.vue`
+Ce composant doit contenir un composant ModalList (crée ci-dessus) qui sera actif ou non selon l'état d'une variable appelée `modal` initialisée a `false`. Le signal `create` reçu par cette ModalList doit être bindé a une fonction `create` qui émettra également un signal `createList` à son parent.
+Il doit également proposer un select basé sur un tableau `lists` reçu en props, et appelant une fonction `changed` lorsque sa valeur change qui va émettre un signal `setCurrentList`. Pour finir, il doit contenir un bouton appelant une fonction `newList` qui passera la variable `modal` a `true` afin de l'activer.
+
+### Menu
+Créez le composant `/src/components/Menu.vue`
+Ce composant doit afficher une checkbox, un bouton, et deux switchs.
+La checkbox va permettre de passer toutes les taches affichées de l'état actif à l'état complété et inversement. Le bouton va permettre de supprimer toutes les taches affichées et complétées. Les deux switchs permettront de choisir d'afficher ou non les taches actives et les taches complétées.
+Ces composants appelleront, sur changement de leur valeur associée, respectivement les fonctions suivantes, qui n'auront pour seul effet que d'émettre un signal a parent. 
+ - `setSelectedTasksComplete`
+ - `deleteSelectedTasks`
+ - `setSeeCompletedTasks`
+ - `setSeeActiveTasks`
+
+### NavBar
+Créez le composant `/src/components/NavBar.vue`
+La barre de navigation ne proposera pour cette étape que la possibilité d'activer ou non les notifications. Elle devra avoir une data `toastsEnabled` qui sera bindée avec le cookie du même nom.
+Lors du changement de valeur de la checkbox, la fonction `setToastEnabled` est appelée et va modifier la valeur du cookie.
+Il est impératif de créer et initialiser ce cookie a `true` dès l'affichage de ce composant dans la fonction `mounted`.
+
+### Task
+Créez le composant `/src/components/Task.vue`
+Ce composant correspond aux taches, il sera appelé via une boucle pour en créer autant que de taches dans notre liste en cours d'édition.
+Il reçoit à sa création les props suivantes: `complete`, `index`, `message`.
```

Comment:
Did you mean "propriétés" ?

Also, "receiving a property" is obscure: are those parameters passed to the componants, or are those its own properties with values set when it's instantiated?

Reply:
Props are specific to vueJs :
'Props are custom attributes you can register on a component. When a value is passed to a prop attribute, it becomes a property on that component instance. '
Should I explain this  ?

Reply:
No, this is my bad: I do not know Vue.js at all.